control-base/src/main/java/fi/nls/oskari/control/feature/DeleteFeatureHandler.java

```diff
@@ -91,6 +93,14 @@ else if (responseString.indexOf("<wfs:totalDeleted>1</wfs:totalDeleted>") > -1)
 			throw new ActionException("Could not update feature");
 		}
 	}
+
+	private int getLayerId(String layerId) throws ActionParamsException {
```

Comment:
This same method seems to be implemented in several different locations with varying error messages. Maybe we should replace them with a single static function?

Reply:
Got a little side-tracked and rewrote the these handlers to some clarity. They should be further updated to use the wfs-t code found in service-myplaces.